src/main/java/io/vertx/cassandra/CassandraClient.java

```diff
@@ -27,19 +27,70 @@
 import io.vertx.core.Vertx;
 
 import java.util.List;
+import java.util.UUID;
 
 /**
  * Eclipse Vert.x Cassandra client.
  */
 @VertxGen
 public interface CassandraClient {
 
+  /**
+   * The name of the default pool
+   */
+  String DEFAULT_POOL_NAME = "DEFAULT_POOL";
+
+
+  /**
+   * Like {@link CassandraClient#create(Vertx, CassandraClientOptions)}  but with default client options
+   */
   static CassandraClient create(Vertx vertx) {
     return create(vertx, new CassandraClientOptions());
   }
 
+  /**
+   * Create a Cassandra client which maintains its own data source.
```

Comment:
`create` method should return a shared client, as recommended by Datastax.

We should either have:
- `create` and `createNonShared`
or
- `createShared` and `createNonShared` methods

Reply:
Let's stick with `createShared` and `createNonShared`